Tests/Factory/DiactorosFactoryTest.php

```diff
@@ -161,4 +161,38 @@ public function testCreateResponseFromBinaryFile()
 
         $this->assertEquals('Binary', $psrResponse->getBody()->__toString());
     }
+
+    public function testUploadErrNoFile()
+    {
+        $file = new UploadedFile(null, null, null, 0, UPLOAD_ERR_NO_FILE, true);
+        $this->assertEquals(0,$file->getSize());
+        $this->assertEquals(UPLOAD_ERR_NO_FILE,$file->getError());
+
+        // SplFile returns false on error
+        $this->assertEquals('boolean',gettype(($file->getSize())));
+        $this->assertFalse($file->getSize());
+
+        // This is an integer, oddly enough internally size is declared as a string
```

Comment:
this comment use useless here as you are not testing UploadedFile here, thus the assertion below is awkward as well